remediation/fence-agents-remediation/configuration.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Configuration
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 2
+---
+
+<!-- markdownlint-disable-next-line MD025 -->
+# Fence Agents Remediation Configuration
+
+Fence Agents Remediation (FAR) uses a Custom Resource (CR) created by [Node Health Check Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (NHC) based on a FAR's remediation template (*FenceAgentsRemediationTemplate*).
```

Comment:
Elsewhere you always mention that FAR CRs can be created manually. And here you just mention NHC.
What about using 2 sections here, one for FAR CRs, and one about the templates with NHC?